### DISCUSSION TITLE
I wanna know what tools were used to test and verify the performance of Qwen2.5-72B-Instruct under VLLM?

### DISCUSSION THREAD

**Discussion by Devliang24** [QwenLM/Qwen3#1054] [Q&A]
addr：https://qwen.readthedocs.io/en/latest/benchmark/speed_benchmark.html

![image](https://github.com/user-attachments/assets/c7cfd3e4-c005-46d1-914f-ba4e55d0933d)

**Comment by wangxingjun778** [comment]
https://github.com/QwenLM/Qwen2.5/pull/1068
